exalearn-api/ExaLearn.Bl/DTO/TopicQuestionDTO.cs

```diff
@@ -6,8 +6,6 @@ public class TopicQuestionDTO
     {
         public int Id { get; set; }
 
-        public LevelType LevelType { get; set; }
```

Comment:
when you create topics level not needed?

Reply:
yes, i talk about this with front team